src/helpers/verifyToken.js

```diff
@@ -0,0 +1,10 @@
+import jwt from 'jsonwebtoken';
+
+const verifyToken = (token) => jwt.decode(token, process.env.JWT_KEY, (err, decoded) => {
+  if (err) {
+    return err;
+  }
+  return decoded;
+});
+
+export default verifyToken;
```

Comment:
You have really done a nice job, I am sure this is the file that makes the coverage drop, Once you correct it, We will be good to go.

Reply:
@hezronkimutai You are right! The error was I used `jwt.decode` instead `jwt.verify` with verify we can catch errors like token  expiration, invalid token, secret key mismatch token as opposed to use decode where we read only data.